src/pages/SignIn/SignIn.js

```diff
@@ -1,34 +1,104 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router';
+import { SIGNIN_API } from '../../config';
+import { KAKAO_API } from '../../config';
```

Comment:
```suggestion
import { SIGNIN_API, KAKAO_API } from '../../config';
```
같은 파일에서 가져오는거는 한번에 import 해주세요

Reply:
넵